.gitignore

```diff
@@ -9,6 +9,7 @@
 target
 /core
 /bd
```

Comment:
You should remove the `bd`-folder, because the name has been changed to `compose`.